src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -402,6 +402,21 @@ internal bool IsLibraryLoaded(string library)
             return importedFunctionGroups.ContainsKey(library);
         }
 
+        /// <summary>
+        /// Checks if a given function is in the builtinFunctionGroups so we do not necessarily look for it's library.
+        /// </summary>
+        /// <param name="library">assembly name</param>
+        /// <param name="nickname">nick name, used for searching as key with default value ""</param>
+        /// <returns></returns>
+        internal bool IsFunctionBuiltIn(string library, string nickname = "")
+        {
+            // For Nodes with Assembly 
+            if (library == "BuiltIn" || library == "Operators")
```

Comment:
Rather than unify the uses, I could not find where these two are originally defined :(